assets/src/edit-story/components/canvas/layout.js

```diff
@@ -38,7 +38,23 @@ import useCanvas from './useCanvas';
  */
 
 const MENU_HEIGHT = 48;
-const MIN_CAROUSEL_HEIGHT = 80;
+
+export const CAROUSEL_VERTICAL_PADDING = 24;
+export const COMPACT_CAROUSEL_VERTICAL_PADDING = 32;
+
+export const COMPACT_THUMB_SIZE = [48, 4];
```

Comment:
For readability reasons I'd split this into two vars, `COMPACT_THUMB_WIDTH` and `COMPACT_THUMB_HEIGHT`. Same goes for the ones below.

Also, since the size is exported, let's make use of it in `CompactIndicator`.